### ISSUE TITLE
Left unary operator

### ISSUE THREAD

**Issue by xLukasbx** [uklimaschewski/EvalEx#126]
Hi,
We can used to unary operator to right side example 5% at the moment? I see in tests that we can used to unary operator but to left side for instance -5.

**Comment by uklimaschewski** [comment]
No, this is not possible. Unary operators are only prefix operators.

**Comment by xLukasbx** [comment]
I think that it is good feature that improvement this library about unary operator on right side in the future.

**Comment by uklimaschewski** [comment]
I marked this as an enhancement.

**Comment by xLukasbx** [comment]
Any have idea how start add feature right side unary operator? I must admit that it is a little difficult for me. I will be grateful for any instruction.

**Comment by uklimaschewski** [comment]
This is probably not so easy. Without knowing, if this is the best or right way:
1) The UnaryOperator class needs a property, if it is postfix or prefix.
2) The parser needs to be adopted to understand the new syntax
3) The shunting yard has to be extended to hanlde a postix unary operator.

**Comment by uklimaschewski** [comment]
Fixed with #257
